app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -247,8 +247,8 @@ def server_status
                  :rulesSource => candlepin_response['rulesSource'],
                  :rulesVersion => candlepin_response['rulesVersion'],
                  :standalone => candlepin_response['standalone'],
-                 :timeUTC => Time.now.getutc,
-                 :version => Katello::VERSION }
+                 :timeUTC => candlepin_response['timeUTC'],
+                 :version => candlepin_response['version'] }
```

Comment:
Thanks @lfu !

Your question about the combined reporting made me look at this API as a whole. If you remember what we talked about earlier: this controller basically proxies directly to Candlepin and injects any extra Katello behavior that's needed (Such as adding the `combined_reporting` capability)

I think we can really simplify this code since everything else is copied directly from the `candlepin_response`.

```
    def server_status                                                                                                                                                                                                                         
      candlepin_response = Resources::Candlepin::CandlepinPing.ping                                                                                                                                                                           
      candlepin_response[:managerCapabilities] << 'combined_reporting'                                                                                                                                                                        
      render :json => candlepin_response                                                                                                                                                                                                      
    end
```

If you compare the output between your dev end and Red Hat Subscription Management you'll see that the results are nearly identical (as they should be) with the primary exception being the combined_reporting capability that was added:

(btw version shows '4.0.0' for be because I am testing Candlepin 4.0.0 in my environment - that's not the Katello version)

API per my recommended changes:
```
{
  "mode": "NORMAL",
  "modeReason": null,
  "modeChangeTime": null,
  "result": true,
  "version": "4.0.0",
  "rulesVersion": "5.41",
  "release": "1",
  "standalone": true,
  "timeUTC": "2021-03-26T18:07:34+0000",
  "rulesSource": "database",
  "managerCapabilities": [
    "instance_multiplier",
    "derived_product",
    "vcpu",
    "cert_v3",
    "hypervisors_heartbeat",
    "remove_by_pool_id",
    "syspurpose",
    "storage_band",
    "cores",
    "hypervisors_async",
    "org_level_content_access",
    "guest_limit",
    "ram",
    "batch_bind",
    "combined_reporting"
  ]
}
```

API from RHSM: `curl -k https://subscription.rhsm.redhat.com/subscription/status | jq .`

```
{
  "mode": "NORMAL",
  "modeReason": null,
  "modeChangeTime": null,
  "result": true,
  "version": "3.2.11",
  "rulesVersion": "5.41",
  "release": "1",
  "standalone": false,
  "timeUTC": "2021-03-26T18:08:11+0000",
  "rulesSource": "default",
  "managerCapabilities": [
    "instance_multiplier",
    "derived_product",
    "vcpu",
    "cert_v3",
    "hypervisors_heartbeat",
    "remove_by_pool_id",
    "syspurpose",
    "insights_auto_register",
    "storage_band",
    "cores",
    "hypervisors_async",
    "org_level_content_access",
    "guest_limit",
    "ram",
    "batch_bind",
    "combined_reporting"
  ]
}

```

Reply:
Neat! 👍